Formula/terragrunt.rb

```diff
@@ -13,12 +13,34 @@ class Terragrunt < Formula
   end
 
   depends_on "go" => :build
-  depends_on "terraform"
 
   def install
     system "go", "build", "-ldflags", "-s -w -X main.VERSION=v#{version}", *std_go_args
   end
 
+  def caveats
+    <<~EOS
+      Terragrunt has been installed as
+        #{HOMEBREW_PREFIX}/bin/terragrunt
+
+      Terragrunt requires a version of `terraform` to be in the user's path.
+
+      Teams using terragrunt/terraform need to use the same version of
+      terraform. To prevent accidental version changes via `brew upgrade`, we
+      recommend using `tfenv` to install and manage `terraform` versions:
+        brew install tfenv
+        tfenv install <required terraform version>
+        tfenv use <required terraform version>
+
+      See: https://github.com/tfutils/tfenv
+
+      Terraform can also be installed directly with homebrew.  This is only
+      recommended when NOT working as part of a team (i.e., for solo projects,
+      learning, or experimenting):
+        brew install terraform
```

Comment:
```suggestion
      terragrunt needs terraform to function. You can install it with:
        brew install terraform
```

Reply:
Hi @carlocab, with respect to homebrew, terraform is similar to ruby.  It's possible to install ruby via homebrew, but there are reasons why most ruby devs use `rbenv` to manage their rubies.  I.e., they need to use different ruby versions for different projects, and they need to upgrade the versions on a different schedule than when new versions appear in homebrew.

Likewise, most devs using terraform (and terragrunt) are probably managing multiple projects, and need explicit control over when terraform versions are used.  Therefore, would you consider something like this:

```
terragrunt needs terraform to function.  You can install it with `tfenv`
(preferred for advanced users), or with:
  brew install terraform
```

Many thanks!

Reply:
I'd rather there were no caveat at all, to be honest.